packages/ses/src/compartment-shim.js

```diff
@@ -198,6 +366,7 @@ export const makeCompartmentConstructor = (
     }
 
     const knownScopeProxies = new WeakSet();
+
     privateFields.set(this, {
```

Comment:
Wow, that's a lot of fields! No change suggested. I don't see any way to simplify. Just sayin ;)

Also curious what other state would be in the proposal that are beyond what the shim can implement. Like those having to do with direct eval.